commands/user.go

```diff
@@ -491,6 +500,36 @@ func updateUserEmailCmdF(c client.Client, cmd *cobra.Command, args []string) err
 	return nil
 }
 
+func updateUserNameCmdF(c client.Client, cmd *cobra.Command, args []string) error {
```

Comment:
Thank you @Aeiyko 

I think we should also add a case to cover the 'invalid username' and one to cover a 'non-existent' username.